cosmos/dbt/project.py

```diff
@@ -12,9 +12,13 @@
 def create_symlinks(project_path: Path, tmp_dir: Path, ignore_dbt_packages: bool) -> None:
     """Helper function to create symlinks to the dbt project files."""
     ignore_paths = [DBT_LOG_DIR_NAME, DBT_TARGET_DIR_NAME, "profiles.yml"]
+
     if ignore_dbt_packages:
-        # this is linked to dbt deps so if dbt deps is true then ignore existing dbt_packages folder
-        ignore_paths.append("dbt_packages")
+        # Check for the presence of 'packages.yml' in the project directory:
+        if (project_path / "packages.yml").exists():
```

Comment:
Let's avoid adding strings representing constants in the code's middle. Could you add `packages.yml` to the `constants.py` file, similar to the other constants? E.g.:
https://github.com/astronomer/astronomer-cosmos/blob/main/cosmos/constants.py#L14

Also, why not have both checks in the same line?
```
if ignore_dbt_packages and (project_path / "packages.yml").exists():
```